libraries/AP_InertialSensor/AP_InertialSensor_MPU9250.cpp

```diff
@@ -458,10 +458,13 @@ void AP_InertialSensor_MPU9250::_accumulate_fast_sampling(uint8_t *samples, uint
             abs(a.z) > clip_limit) {
             clipped = true;
         }
-        asum += a;
-        gsum += Vector3l(int16_val(data, 5),
```

Comment:
_asum_ and _gsum_ aren't needed anymore right? You removed the declaration from MPU6000 but not here (GitHub doesn't allow me to do the comment in the right place unfortunately).